guides/release/configuring-ember/debugging.md

```diff
@@ -106,7 +106,7 @@ Ember = {
 };
 ```
 
-#### Errors within an `RSVP.Promise`
+### Errors within an `RSVP.Promise`
```

Comment:
I think this is “fine”, but _in general_ folks shouldn’t need this anymore.

Reply:
Removed for simplicity!